docs/docs-source/README.adoc

```diff
@@ -4,21 +4,117 @@ The source files for this guide use asciidoc syntax and the project is configure
 
 This readme covers the following topics:
 
+
+* <<Support for multiple versions>>
 * <<Building the guide locally>>
 * <<Adding tabbed example blocks>>
 * <<Linking to section headings>>
 * <<Versioning the doc>>
 
+== Support for multiple versions
+
+The documentation structure and supporting scripts is set up to allow the easy evolution of versioned documentation alongside the main project.
+Each version is maintained in a separate branch that steems from the release that generated the new version.
+For example, for `v1.3.3` of the documentation, there's a (`v1.3.3-docs`)[https://github.com/lightbend/cloudflow/tree/v1.3.3/] branch that contains the version of the documents and links to the examples that corresponds to the Cloudflow 1.3.3 release.
```

Comment:
The tree link says v1.3.3 but branch says v1.3.3-docs?

Reply:
ah read further down, maybe say there is a v1.3.3 branch that is the version for release purposes, and v1.3.3-docs for docs purposes? Or am I confused?

Reply:
my bad. fixed!